packages/alfa-wcag/scripts/techniques.js

```diff
@@ -0,0 +1,49 @@
+const fs = require("fs");
+const path = require("path");
+const prettier = require("prettier");
+const puppeteer = require("puppeteer");
+
+puppeteer.launch().then(async (browser) => {
+  const page = await browser.newPage();
+
+  await page.goto("https://www.w3.org/WAI/WCAG21/Techniques/");
```

Comment:
I guess we might need 2.2 techniques at some point 🤔
If there's no technique that disappeared, we should probably only grab 2.2 techniques.

Reply:
The WCAG 2.2 techniques are currently the same as the 2.1 techniques, sans all the Flash techniques: https://www.w3.org/WAI/WCAG22/Techniques/